backend/src/main/java/org/polsl/backend/service/HardwareService.java

```diff
@@ -57,14 +56,16 @@ public HardwareService(HardwareRepository hardwareRepository,
     this.affiliationHardwareRepository = affiliationHardwareRepository;
   }
 
-  public PaginatedResult<HardwareListOutputDTO> getHardwareList(boolean soloOnly, Specification<Hardware> spec) {
+  //region HTTP METHODS
+  public PaginatedResult<HardwareListOutputDTO> getHardwareList(Specification<Hardware> spec) {
     Iterable<Hardware> hardwareList;
```

Comment:
Zmienną możesz zadeklarować później, wraz z jej inicjalizają

Reply:
Albo, zrobisz to kiedyś przy okazji 😛

Reply:
Poprawiłem w #165 😄